### PR TITLE
grpc: add more information for grpc pool

### PR DESCRIPTION
**Pull request by bufferflies**

### CODE REVIEW

File: proto/kvrpcpb.proto [hunk 1]
```diff
@@ -1095,6 +1095,9 @@ message TimeDetailV2 {
     uint64 kv_read_wall_time_ns = 4;
     // Total wall clock time spent on this RPC in TiKV .
     uint64 total_rpc_wall_time_ns = 5;
+
+    uint64 grpc_exec_time_ns = 6;
```
Comment (nolouch) [inline proto/kvrpcpb.proto:1099]:
```suggestion
    uint64 kv_grpc_pool_exec_time_ns = 6;
```

File: proto/kvrpcpb.proto [hunk 2]
```diff
@@ -1095,6 +1095,9 @@ message TimeDetailV2 {
...
+    uint64 grpc_wait_time_ns = 7;
```
Comment (nolouch) [inline proto/kvrpcpb.proto:1100]:
```suggestion
    uint64 kv_grpc_pool_wait_time_ns = 7;
```

File: proto/kvrpcpb.proto (hunk 1 above)
Comment (glorv) [reply proto/kvrpcpb.proto:1099]: please also add a comment for the new fields.

**Review by glorv — APPROVED** [review]

**Comment by glorv** [comment]
Please run `make check` to update protolock

**Review by nolouch — APPROVED** [review]

**Pull request merged**